testing/internal/e2e/tests/base/target_tcp_connect_exec_long_lasting_script_test.go

```diff
@@ -86,34 +85,25 @@ func TestCliTcpTargetConnectExecLongLastingScript(t *testing.T) {
 	require.NoError(t, output.Err, string(output.Stderr))
 
 	// Send SSH requests to the target to execute long-lasting scripts
-	var eg errgroup.Group
-	eg.Go(func() error {
-		t.Log("Executing the long-lasting script saved on the target...")
-		cmd := exec.CommandContext(ctx, "/usr/bin/ssh",
-			"-v",
-			"-l", c.TargetSshUser,
-			"-i", c.TargetSshKeyPath,
```

Comment:
Doesn't this command need credentials for it to work?

Reply:
Good catch! Got it back 👍